submissions/morenko/building-a-tiny-js-world/index.js

```diff
@@ -0,0 +1,43 @@
+const woman = {
+    species: 'human',
+    name: 'line',
+    gender: 'female',
+    legs: 2,
+    hands: 2,
+    saying: 'bye',
+  };
+const man = {
+    species: 'human',
+    name: 'Joey',
+    gender: 'male',
+    legs: 2,
+    hands: 2,
+    saying: 'hi',
+  };
+const dog = {
+    species: 'dog',
+    name: 'Dik',
+    gender: 'male',
+    legs: 4,
+    hands: 0,
+    saying: 'woof',
+  };
+const cat = {
+    species: 'cat',
+    name: 'Tom',
+    gender: 'male',
+    legs: 4,
+    hands: 0,
+    saying: 'mau',
+  };
+  
+  
+  const inhabitants = [dog, cat, woman, man,];
```

Comment:
There is no reason to leave a trailing coma if a whole array is on the same line.